runtime/nonfb/FBLocaleToLang.js

```diff
@@ -52,8 +52,13 @@ const _locToLang = {
 
 const FBLocaleToLang = {
   get: function(locale) {
-    return _locToLang[locale] || locale.substring(0, locale.indexOf('_'));
-  },
+    const lodashIndex = locale.indexOf("_");
+
+    return (
```

Comment:
nit: ditch the parens

Reply:
I am not an English speaker and the next phrase untranslatable

> ditch the parens

Reply:
Don't worry - I'll fix this up internally when landing.  I was asking to remove the parentheses.  This code is actually generated, so we'll need to fix the internal script anyhow.